src/map/lapack2flash/FLASH_lapack2flash_util.c

```diff
@@ -245,53 +241,46 @@ int FLAME_QR_piv_preorder( FLA_Obj A, int *jpiv_lapack, int *jpiv_fla )
 }
 */
 
-/*---------------------Get/Set Helper---------------------*/
-
-FLA_Error FLASH_set_preferred_blocksize( dim_t blocksize )
-{
-    flash_blocksize = blocksize;
-
-    return FLA_SUCCESS;
-}
-
-dim_t FLASH_get_preferred_blocksize( void )
+FLA_Bool FLASH_Check_Offload_to_GPU( dim_t blocksize, int m, int n, unsigned int offset )
```

Comment:
thinking about whether this routine should just call FLASH_get_tile_offload() instead of having it be passed in?

Reply:
I thought about that, but opted for a pass in so we could tune each differently if we wanted to. 
I.e., give chol a 'FLASH_get_tile_offload() + 1' and have LU just pass in 'FLASH_get_tile_offload()' if we found that to be best while tuning. This may not be necessary.